packages/button/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.2.0",
   "main": "dist/main.js",
   "license": "MIT",
+  "typings": "definitions/index.d.ts",
```

Comment:
this is up to discussion, because current build is non existent till https://github.com/wc-catalogue/blaze-elements/pull/153